app/helpers/index.js

```diff
@@ -0,0 +1,15 @@
+const { userSchema } = require('./userSchema');
```

Comment:
Instead of having schemas in helpers, you could have a folder with all schemas of your API